packages/jsii-pacmak/test/expected.jsii-calc/java/src/main/java/software/amazon/jsii/tests/calculator/Calculator.java

```diff
@@ -266,4 +208,54 @@ public void setUnionProperty(final software.amazon.jsii.tests.calculator.Multipl
     public void setUnionProperty(final software.amazon.jsii.tests.calculator.Power value) {
         this.jsiiSet("unionProperty", value);
     }
+
+    /**
+     * A fluent builder for {@link software.amazon.jsii.tests.calculator.Calculator}.
+     */
+    @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+    public static final class Builder {
+        /**
+         * EXPERIMENTAL
+         * 
+         * @return a new instance of {@link Builder}.
+         */
+        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+        public static Builder create() {
+            return new Builder();
+        }
+
+        private software.amazon.jsii.tests.calculator.CalculatorProps.Builder props;
+
+        private Builder() {
+        }
+
+        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+        public Builder initialValue(final java.lang.Number initialValue) {
+            this.props().initialValue(initialValue);
+            return this;
+        }
+
+        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+        public Builder maximumValue(final java.lang.Number maximumValue) {
+            this.props().maximumValue(maximumValue);
+            return this;
+        }
+
+        /**
+         * @returns a newly built instance of {@link software.amazon.jsii.tests.calculator.Calculator}.
+         */
+        @software.amazon.jsii.Stability(software.amazon.jsii.Stability.Level.Experimental)
+        public software.amazon.jsii.tests.calculator.Calculator build() {
+            return new software.amazon.jsii.tests.calculator.Calculator(
+                this.props != null ? this.props.build() : null
+            );
```

Comment:
This is more readable but not what I actually meant about multi-line. My apologies for not being explicit and clear. I think it should be split into a local variable assignment followed by passing that local variable in as an argument. This is more of a nit though so I wouldn't block the build on it, it's your call.